lib/SensorFusion/src/SerialMuxChannels.h

```diff
@@ -42,15 +43,21 @@
  * Macros
  *****************************************************************************/
 
-/** Maximum number of SerialMuxProt Channels. */
-#define MAX_CHANNELS (10U)
-
 /** Name of Channel to send Sensor Data to. */
 #define SENSORDATA_CHANNEL_NAME "SENSOR_DATA"
 
 /** DLC of Sensordata Channel */
 #define SENSORDATA_CHANNEL_DLC (sizeof(SensorData))
 
+/** Name of Channel to End Line Detected Signal. */
+#define ENDLINE_CHANNEL_NAME "END_LINE"
+
+/** DLC of End Line Channel */
+#define ENDLINE_CHANNEL_DLC (sizeof(EndLineFlag))
+
+/** Maximum number of SerialMuxProt Channels. */
+#define MAX_CHANNELS (10U)

```

Comment:
Why moved to the bottom?
It is usually the first defined constant in the file. Not a problem per-se, but we should remain consisten across applications